docs/import/manage_layers.js

```diff
@@ -328,31 +327,10 @@ function populateStateAndDisasterAssetPickers(disaster) {
   for (const state of states) {
     if (!stateAssets.has(state)) statesToFetch.push(state);
   }
-  // TODO: add functionality to re-pull all cached states from ee without
-  // reloading the page.
-  if (statesToFetch.length === 0) {
-    createStateAssetPickers(states);
-  } else {
-    const statesDone = getStatesAssetsFromEe(statesToFetch).then((assets) => {
-      for (const asset of assets) {
-        stateAssets.set(asset[0], asset[1]);
-      }
-      createStateAssetPickers(states);
-    });
-    promises.push(statesDone);
-  }
 
   return Promise.all(promises);
 }
 
-/**
- * Create asset pickers for the given states.
- * @param {Array<string>} states of the form ['WA', ...]
- */
-function createStateAssetPickers(states) {
-  createAssetPickers(states, stateAssets, $('#state-asset-pickers'));
-}
-
 /**
  * Create an asset picker for the given disaster.
  * @param {string} disaster
```

Comment:
could fold createDisasterAssetPicker and createAssetPickers into one function now since the former is the only caller of the latter (unless you think we might need it in the future).

Reply:
I'm actually doing the folding in my disable-empty-geometry PR so all good here